package-lock.json

```diff
@@ -219,6 +219,7 @@
       "resolved": "https://registry.npmjs.org/@algolia/client-search/-/client-search-5.37.0.tgz",
       "integrity": "sha512-DAFVUvEg+u7jUs6BZiVz9zdaUebYULPiQ4LM2R4n8Nujzyj7BZzGr2DCd85ip4p/cx7nAZWKM8pLcGtkTRTdsg==",
       "license": "MIT",
+      "peer": true,
```

Comment:
Why and how was this added ?

Reply:
uggg, sorry!! being lazy. told AI do it for me